multimodal/src/autogluon/multimodal/utils/onnx.py

```diff
@@ -60,15 +60,21 @@ def __init__(self, onnx_path: str, providers: Optional[Union[dict, List[str]]] =
         import onnx
         import onnxruntime as ort
 
-        if not os.path.exists(onnx_path):
-            raise FileNotFoundError(f"failed to located onnx file at {onnx_path}")
+        if isinstance(onnx_path, bytes):
+            onnx_model = onnx.load_model_from_string(onnx_path)
+        else:
+            if not os.path.exists(onnx_path):
+                raise FileNotFoundError(f"failed to located onnx file at {onnx_path}")
 
-        logger.info("Loading ONNX file from path {}...".format(onnx_path))
-        onnx_model = onnx.load(onnx_path)
+            logger.info("Loading ONNX file from path {}...".format(onnx_path))
+            onnx_model = onnx.load(onnx_path)
 
         if providers == None:
```

Comment:
use `if providers is None` as a better practice: https://stackoverflow.com/questions/3257919/what-is-the-difference-between-is-none-and-none

Reply:
updated accordingly